src/components/History/History.tsx

```diff
@@ -48,67 +48,69 @@ const History = ({ onBackButtonClick }: { onBackButtonClick: () => void }) => {
         const assets = await getSupportedAssetsForChainId(chain_id);
 
         value.map((item) => {
-          const assetnetwork =
-            item.asset && assets ? assets.find((supportedAsset) => supportedAsset.symbol == item.asset.symbol) : null;
-          const chainId = chain_id;
-          const crossChainActionId = uniqueId(`${item.timestamp}-`);
-          const receiverAddress = item.asset ? item.asset.to : item.to;
-          const fromAddress = item.asset ? item.asset.from : item.from;
-          const isFromEtherspotWallet = false;
-          const transactionUrl = item.blockExplorerUrl;
-
-          let estimated = {
-            usdPrice: UsdPrice,
-            gasCost: item.gasPrice,
-            feeAmount: item.asset && item.asset != null && item.asset.value ? item.asset.value : null,
-          };
-          let preview = {
-            chainId,
-            receiverAddress,
-            fromAddress,
-            isFromEtherspotWallet,
-            transactionUrl,
-            asset: {
-              address: item.asset ? item.asset.from : item.from,
-              decimals: item.asset ? item.asset.decimal : 2,
-              symbol: item.asset ? item.asset.symbol : '',
-              amount: '100000000000000000',
-              iconUrl: assetnetwork != null ? assetnetwork?.logoURI : '',
+          if (item.asset != null) {
```

Comment:
```suggestion
          if (!item?.asset) return;
```

and then no need to wrap rest of the code inside operator brackets